test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -154,6 +154,18 @@ def test_vol(self, gain, gain_type):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
 
+    @parameterized.expand([
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': True, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': False}, ),
```

Comment:
this set of params is a duplicate of the first one -- could you change or remove it?

Reply:
Thanks for catching that, will change it.

Reply:
Removed the duplicate test case. Thanks!